src/controllers/UserController.ts

```diff
@@ -49,6 +49,21 @@ export class UserController {
 		}
 	}
 
+	public async resendEmail(req: Request, res: Response, next: NextFunction): Promise<void> {
+		try {
+			const user = await this.userService.resendEmail(req.body);
```

Comment:
Instead of `req.body`, could you use `req.body.email` as the body should look like this:

```js
{
  "email": "blah@manchester.ac.uk
}
```